README.md

```diff
@@ -21,7 +19,7 @@ https://github.com/kubernetes/dashboard/wiki/Access-control) guide before perfor
 To deploy Dashboard, execute following command:
 
 ```sh
-$ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/aio/deploy/recommended/kubernetes-dashboard.yaml
```

Comment:
I have sent PR to the AWS docs. Hopefully it will be resolved soon.

Wiki is fixed.

However, we cannot ensure that articles posted in the past, not even by us will stay always up to date.